### ISSUE TITLE
1Password extension conflicts with `navigator.credentials.get`

### ISSUE THREAD

**Issue by brianorwhatever** [digitalbazaar/credential-handler-polyfill#38]
Starting around version 2.14.0 the 1Password extension is capturing all of the `get` requests and not passing them onto other scripts such as CHAPI. This is likely bad behaviour on their behalf..

**Comment by msporny** [comment]
We are starting to hear from more people in the field that are hitting this issue. We really need to push a fix to detect and re-patch (or something equivalent). Let's get back in touch w/ the 1Password folks to see what their plans are to ensure this doesn't happen in future releases... and in parallel, ship code to detect and fix (if possible) the issue .

**Comment by Custard7** [comment]
We just started seeing errors related to the Credential Management API: CHAPI is breaking for us when the browser already has `navigator.credentials.get()`

https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/get

Have y'all started to run into this?

**Comment by dlongley** [comment]
Yes, #42 is the proposed fix for this -- we just need to do some more testing and then deploy it. cc: @jameseaster

**Comment by jameseaster** [comment]
I've tested #42 and it looks good to go!

**Comment by davidlehn** [comment]
3.2.1 was released with the update.  Please test in various environments.  If no further reports come in, this can be closed.
